plasmapy/physics/tests/test_quantum.py

```diff
@@ -71,26 +61,29 @@ def test_thermal_deBroglie_wavelength():
     r"""Test the thermal_deBroglie_wavelength function in quantum.py."""
     lambda_dbTh = thermal_deBroglie_wavelength(T_e)
     # test a simple case for expected value
+    expectStr = "Thermal deBroglie wavelength is off expected result for 1eV"
     assert np.isclose(lambda_dbTh.value,
-                      6.919367518364532e-10)
+                      6.919367518364532e-10), expectStr
     # testing returned units
     assert lambda_dbTh.unit == u.m
 
 def test_Fermi_energy():
     r"""Test the Fermi_energy function in quantum.py."""
     energy_F = Fermi_energy(n_e)
     # test a simple case for expected value
+    expectStr = "Fermi energy is off expected result for 1e23 cm^-3"
```

Comment:
Actually, is this an `expected value string` or an `exception string`?

Reply:
Well it's telling you that the expected value is off, but it's thrown as an exception, so both? 😖

Reply:
Hah :smile: